ocaml/libs/uuid/dune

```diff
@@ -3,8 +3,13 @@
   (public_name uuid)
   (modules uuidx)
   (libraries
-    unix (re_export uuidm)
+    mtime
+    mtime.clock.os
```

Comment:
Note that both `{ptime,mtime}.clock.os` are deprecated use `{ptime,mtime}.os` if you are using the latest version (also clocks are made to re-export the base package so you can also specify only those if you wish so). See https://discuss.ocaml.org/t/ann-ptime-1-2-0-mtime-2-1-0-qrc-0-2-0/15268